### DISCUSSION TITLE
Collision in intersection

### DISCUSSION THREAD

**Discussion by RuiFilipe2802** [eclipse/mosaic#334] [Q&A]
Hello,

I want to get warnings for collisions taking place in a intersection as shown

![image](https://github.com/eclipse/mosaic/assets/56730143/aec055a2-6ab6-4e8b-9cfe-5e347409349b)

I am using "--collision.action warn --collision.check-junctions true" as "additionalSumoParameters" but still no collision is detected on the intersection. Do you have any idea on what could be the problem? I already asked on the sumo GitHub but as I found out the problem is not from sumo per say, since I created a similar scenario only using sumo and the collisions were detected.

Thanks in advance!

**Comment by kschrab** [comment]
Hi, if you could give us your scenairo we could have a look into it. Either by attaching it here or send to mosaic@fokus.fraunhofer.de

**Reply by RuiFilipe2802** [reply]
Thanks, I've sent it by email 🙂
